src/status_im2/contexts/quo_preview/list_items/preview_lists.cljs

```diff
@@ -104,10 +104,9 @@
 
 (defn view
   []
-  (let [state (reagent/atom {:type               :accounts
-                             :size               :size/s-32
-                             :number             4
-                             :more-than-99-label "99+"})
+  (let [state (reagent/atom {:type   :accounts
+                             :size   :size/s-32
```

Comment:
again, about `s-32`, why not `m-32` or `l-32`? do we have a rule to know when to use each letter?

Reply:
let's use `:size-32` - see discussion and result here:
https://github.com/status-im/status-mobile/pull/17279

btw the idea was to always use `-s` as in `-size`, so clearly that idea was confusing 😆